### ISSUE TITLE
How to use the pretrain model 

### ISSUE THREAD

**Issue by ankan8145** [lambert-x/medical_mae#7]
Can anyone tell me how to use the checkpoint of "ViT-Base/16" model ? and what is mask ratio here mask_ratio=1 means full mask the full or mask_ratio=0 means full mask the full image ?

**Comment by SaumyaBhandari** [comment]
mask_ratio=1 means full mask, as the paper mentions that they have used 95% masking, which is termed as 0.95 mask_ratio. 

You can look at my closed issue [here](https://github.com/lambert-x/medical_mae/issues/6) where I've given the code to seamlessly load the weights from the pretrained model. 

Just include the filepath of the checkpoint in args.finetune.
